src/Application/Apps/Commands/CreateAppCommandValidator.cs

```diff
@@ -34,4 +38,11 @@ public async Task<bool> BeUniqueName(CreateAppCommand model, string name, Cancel
     {
         return await _context.Apps.AllAsync(a => a.Name != name, cancellationToken);
     }
+
+    public async Task<bool> BeValidStorageId(CreateAppCommand model, string storageId, CancellationToken cancellationToken)
```

Comment:
can we change this to `ExistInBindleServer` so we can differentiate this from `validStorageId`?